grizli/prep.py

```diff
@@ -4716,15 +4916,16 @@ def visit_grism_sky(grism={}, apply=True, column_average=True, verbose=True, ext
                         label=grism['files'][j].split('_fl')[0])
 
         # result
-        fp = open(file.replace('_flt.fits', '_column.dat'), 'wb')
+        fp = open(file.replace('_flt.fits', '_column.dat').replace('_rate.fits', '_column.dat'), 'wb')
         fp.write(b'# column obs_resid ok resid uncertainty\n')
         np.savetxt(fp, np.array([xmsk, yres, yok*1, y_pred-bg_sky, gp_sigma]).T, fmt='%.5f')
         fp.close()
 
         if apply:
             # Subtract the column average in 2D & log header keywords
-            gp_res = np.dot(y_pred[:, None]-bg_sky, np.ones((1014, 1)).T).T
             flt = pyfits.open(file, mode='update')
+            imshape = flt[1].data.shape[0]
```

Comment:
Are we sure that the JWST grism exposures are being rotated correctly in this `prep` processing so that the column subtraction is truly columns parallel to the dispersion axis as expected?

I put some functions to compute the rotations in [grismconf.py](https://github.com/gbrammer/grizli/blob/master/grizli/grismconf.py#L590)

Reply:
I've added a check for rot90: d42e1240cfbd85a60beb56e8237db7d32c19938c

In the cases we've had so far, it has seemed to be ok. However, I'm not totally sure I switched the axes correctly for the case where we'd need to do that.

Reply:
I don't see it in the diff here, but you'd need to change the axes along with the statistics are averaged [here](https://github.com/gbrammer/grizli/blob/master/grizli/prep.py#L4626).